resources/lib/telstra_auth.py

```diff
@@ -0,0 +1,86 @@
+#-------------------------------------------------------------------------------
+import requests
+import collections
+import json
+import urlparse
+import config
+import comm
+import utils
+from exception import AFLVideoException
+
+class SortedHTTPAdapter(requests.adapters.HTTPAdapter):
+    def add_headers(self, request, **kwargs):
+        if 'header_order' in globals():
```

Comment:
Just remembered that only the NRL auth process needed the specific header orders and even now it seems they've relaxed that strange requirement so I'll just remove altogether.